infrastructure/resource_group_level/modules/front-door-tfm.bicep

```diff
@@ -140,3 +140,4 @@ resource frontDoorTfm 'Microsoft.Network/frontdoors@2021-06-01' = {
   }
 }
 
+output defaultHostName string = frontDoorTfm.properties.frontendEndpoints[0].properties.hostName
```

Comment:
Note that this will correspond to the default value of `tfs-feature=tfm-fd.azurefd.net` and not any custom hostname. I've checked `prod` and it uses the equivalent url.